config/locales/metadata/en.yml

```diff
@@ -61,6 +61,9 @@ en:
         number_of_samples_per_pool:
```

Comment:
do we still need number of samples per pool defined?

Reply:
Hmm, probably not. Going to leave for now as I can't really test if removing it breaks anything (laptop setup...)